src/OpenSage.Game/Logic/Object/Draw/W3dModelDraw.cs

```diff
@@ -120,6 +120,11 @@ protected virtual bool SetActiveAnimationState(AnimationState animationState, Ra
                 return false;
             }
 
+            if (animationState?.Script is not null)
```

Comment:
i'd prefer `!= null`. I don't think we use the other notation anywhere

Reply:
It is used elsewhere (11 usages in 7 files), but not as prevalent as `!= null`. The same goes for `is null` vs `== null`. I'm fine using `!= null` if that's the preferred style :)